### ISSUE TITLE
/tebex lookup - better end user feedback.

### ISSUE THREAD

**Issue by andrewkm** [tebexio/Tebex-Minecraft#12]
Paper 1.20.4 Build 389
Tebex-Minecraft v2.0.1 compiled against: https://github.com/tebexio/Tebex-Minecraft/commit/da5c972f8998341476f7f262f2eb5fd85a95ba7c

`/tebex lookup <regular known player>`
- **Works**
- Example: `/tebex lookup videowiz92`
- https://i.imgur.com/KbMSjjZ.png

`/tebex lookup <player who hasn't joined your server and perhaps hasn't used tebex>`
- Example `/tebex lookup Johnson1`
- https://i.imgur.com/yLZJl9z.png

`/tebex lookup <player who is banned/chargedback on your server>`
- Example on ours: `/tebex lookup angelicdew`
- https://i.imgur.com/LfQYCey.png
- Although after several tries, it reverts to this error: https://i.imgur.com/yLZJl9z.png

TL;DR errors/failed attempts should be coherent and make sense to the end user.

**Comment by andrewkm** [comment]
Not fixed, yet closed.
